python/ql/test/experimental/semmle/python/frameworks/clickhouse-driver/ClickHouseDriver.ql

```diff
@@ -0,0 +1,6 @@
+import python
+import experimental.semmle.python.frameworks.ClickHouseDriver
+import semmle.python.Concepts
+
+from SqlExecution::Range s
```

Comment:
```suggestion
from SqlExecution s
```

Reply:
I should probably add that `<class>::Range` is for defining new instances, and then you use `<class>` when you actually want to select instances... This is described in more detail in https://github.com/github/codeql/blob/main/docs/ql-design-patterns.md#range-for-extensibility-and-refinement